### PR TITLE
Create proftpd.xml Template

### PR DESCRIPTION
**Pull request by tooliload**

### CODE REVIEW

File: templates/proftpd.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <Config Name="Passive Ports" Target="49152-49407" Default="49152-49407" Mode="tcp" Description="" Type="Port" Display="always" Required="false" Mask="false">49152-49407</Config>
+  <Config Name="FTP_DB_USER" Target="FTP_DB_USER" Default="" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false"></Config>
+  <Config Name="FTP_DB_PASS" Target="FTP_DB_PASS" Default="" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="true"></Config>
+  <Config Name="FTP_DB_HOST" Target="FTP_DB_HOST" Default="localhost" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false"></Config>
+  <Config Name="FTP_DB_NAME" Target="FTP_DB_NAME" Default="proftpd" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false"></Config>
+  <Config Name="MASQ_ADDR" Target="MASQ_ADDR" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="PWD_SALT" Target="PWD_SALT" Default="DockerCrypt" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">SomeCryptPass</Config>
```
Comment (nwithan8) [inline templates/proftpd.xml:29]: Your Default should be the same as the visible text here ("DockerCrypt" != "SomeCryptPass")

File: templates/proftpd.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <Config Name="MOD_VROOT" Target="MOD_VROOT" Default="ON" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">ON</Config>
+  <Config Name="FTP_VERBOSE" Target="FTP_VERBOSE" Default="UNDEF" Mode="" Description="ON/UNDEF, activate/deactivate logging to files" Type="Variable" Display="always" Required="true" Mask="false">ON</Config>
+  <Config Name="VERBOSE" Target="VERBOSE" Default="OFF" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false">ON</Config>
```
Comment (nwithan8) [inline templates/proftpd.xml:32]: Default "OFF", but text is "ON". Make them consistent.

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/proftpd.xml (hunk 2 above)
Comment (tooliload) [reply templates/proftpd.xml:32]: new pull to #419

**Comment by nwithan8** [comment]
Closing in favor of #419

**Pull request closed without merging**